src/main/scala/db/migration/V4__MoveRelatedContentEmbedsToDivs.scala

```diff
@@ -0,0 +1,126 @@
+/*
+ * Part of NDLA draft_api.
+ * Copyright (C) 2018 NDLA
+ *
+ * See LICENSE
+ */
+
+package db.migration
+
+import java.sql.Connection
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration
+import org.json4s.Extraction.decompose
+import org.json4s.JsonAST.JArray
+import org.json4s.native.JsonMethods.{compact, parse, render}
+import org.json4s.{DefaultFormats, JValue}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Element
+import org.jsoup.nodes.Entities.EscapeMode
+import org.postgresql.util.PGobject
+import scalikejdbc.{DB, DBSession, _}
+
+import scala.collection.JavaConverters._
+
+class V4__MoveRelatedContentEmbedsToDivs extends JdbcMigration {
+
+  implicit val formats: DefaultFormats.type = org.json4s.DefaultFormats
+
+  override def migrate(connection: Connection): Unit = {
+    val db = DB(connection)
+    db.autoClose(false)
+
+    db.withinTx { implicit session =>
+      migrateArticles
+    }
+  }
+
+  def migrateArticles(implicit session: DBSession): Unit = {
+    val count = countAllArticles.get
+    var numPagesLeft = (count / 1000) + 1
+    var offset = 0L
+
+    while (numPagesLeft > 0) {
+      allArticles(offset * 1000).map {
+        case (id, document) => updateArticle(convertArticleUpdate(document), id)
+      }
+      numPagesLeft -= 1
+      offset += 1
+    }
+  }
+
+  def countAllArticles(implicit session: DBSession): Option[Long] = {
+    sql"select count(*) from articledata where document is not NULL"
+      .map(rs => rs.long("count"))
+      .single()
+      .apply()
+  }
+
+  def allArticles(offset: Long)(implicit session: DBSession): Seq[(Long, String)] = {
+    sql"select id, document from articledata where document is not null order by id limit 1000 offset ${offset}"
+      .map(rs => {
+        (rs.long("id"), rs.string("document"))
+      })
+      .list
+      .apply()
+  }
+
+  private def convertContent(content: List[V4_Content]): JValue = {
+    val contents = content.map(cont => {
+      val document = Jsoup.parseBodyFragment(cont.content)
+      document
+        .outputSettings()
+        .escapeMode(EscapeMode.xhtml)
+        .prettyPrint(false)
+        .indentAmount(0)
+
+      for (embed <- document.select("embed").asScala) {
```

Comment:
Kanskje `.select("embed[data-resource='related-content']"` for å være helt sikker?

Reply:
Det er nok smart :+1: